SECURITY.md

```diff
@@ -11,8 +11,8 @@
 
 To report a vulnerability, please open an issue or email rdconner@us.ibm.com and taylorjones@us.ibm.com
 
-As this is a UI library, it's highly uncommon to see a security vulnerability directly within this codebase, but it is possible. 
+As this is a UI library, it's highly uncommon to see a security vulnerability directly within this codebase, but it is possible.
 
-If a reported vulnerability is within the codebase, the issue will be added to the current sprint and someone will begin to investigate immediately. Some components/modules we export are a direct proxy of a module from the [Carbon Design System](https://github.com/carbon-design-system/carbon). If the vulnerability is there, a maintainer will contact a member of the Carbon team and we'll work with them to investigate. 
+If a reported vulnerability is within the codebase, the issue will be added to the current sprint and someone will begin to investigate immediately. Some components/modules we export are a direct proxy of a module from the [Carbon Design System](https://github.com/carbon-design-system/carbon). If the vulnerability is there, a maintainer will contact a member of the Carbon team and we'll work with them to investigate.
```

Comment:
This is fine - I wrote this doc in the github interface in the security getting started wizard thing. So your git hook ran prettier which corrected some whitespace